keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ComparableLinksTests.kt

```diff
@@ -147,21 +151,30 @@ class ComparableLinksTests : JUnit5Minutests {
       number = "1",
     )
 
```

Comment:
Because I changed the way we interact with the database (we pull the artifact versions info as one list and the status info as a separate list with just the version) we now get the artifact details from the list of artifact versions. Because of that, I needed to change the way this test was structured. The `git(...)` function here is responsible for making that list of artifacts.

I was able to remove a lot of mocked calls because we are no longer making redundant db calls for every single version. But, because of that, the versions in the test changed because the pinned version became a real version and not just the version string "pinned".